### PR TITLE
Symfony 5.0 compatibility

### PR DESCRIPTION
**Pull request by umpirsky**
Fixes issue #66.

### CODE REVIEW

**Comment by umpirsky** [comment]
@nacmartin If this can't be release soon, please let me know. Thanks.

**Comment by umpirsky** [comment]
@nacmartin Can we merge this please?

File: src/Limenius/Liform/Serializer/Normalizer/FormErrorNormalizer.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface as TranslatorContract;
```
Comment (nacmartin) [inline src/Limenius/Liform/Serializer/Normalizer/FormErrorNormalizer.php:18]: Why are you leaving TranslatorInterface along with TranslatorContract? Why not using only one of the two?

**Comment by nacmartin** [comment]
@umpirsky Hi, actually we got green light to update the project that was using this, so hopefully I can allocate more time to this library.

We were looking at it. I left a comment if you don't mind the question.

File: src/Limenius/Liform/Serializer/Normalizer/FormErrorNormalizer.php (hunk 1 above)
Comment (umpirsky) [reply src/Limenius/Liform/Serializer/Normalizer/FormErrorNormalizer.php:18]: For backward compatibility with the older versions of Symfony.

**Comment by nacmartin** [comment]
Thanks, @umpirsky

**Pull request merged**